public/css/admin.css

```diff
@@ -40,6 +47,7 @@ h3 {
 	position: relative;
 	text-align: center;
 
+	color: white;
 	background-color: white; /*backup*/
```

Comment:
Same thing about CSS variables here too. Even though it's more verbose for the same effect, `var(--white)` is simply better practice. I won't stop and leave a comment on every single line with a color in it, but please go through and make sure you're only using CSS variables